packages/widget/src/widget/initAmplitude.ts

```diff
@@ -13,7 +13,9 @@ export const initAmplitude = () => {
   init("14616a575f32087cf0403ab8f3ea3ce0", {
     autocapture: true,
     appVersion: version,
+    // serverUrl: "https://go.skip.build/api/amplitude"
```

Comment:
nit: should we remove this if we're not using it? typically we dont keep commented out code around

Reply:
Im working on a separate PR on the app side to get this setup. So itll be commented out shortly!